workspace/api/openfdd_bridge/zone_energy_research.py

```
@@ -305,6 +311,20 @@ def _llm_research_tasks(
             "Prioritize opportunities[] topics (energy_setback, sensor_integrity, overnight_load) "
             "in plain English with cautious wording (verify on site)."
         )
+    if fan_schedule:
+        wd = fan_schedule.get("weekday") or {}
+        we = fan_schedule.get("weekend") or {}
+        tasks.append(
+            "Report HVAC supply-fan on/off patterns from fan_schedule: typical weekday first/last fan-on "
+            f"hours (weekday {wd.get('typical_first_fan_on_hour')}–{wd.get('typical_last_fan_on_hour')} local), "
```

Ruff (0.15.15)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)